README.md

```diff
@@ -30,7 +30,7 @@ Requirements
 ------------
 
 - PHP 8.1+
-- [Composer](https://getcomposer.org/)
+- [Composer 2](https://getcomposer.org/)
```

Comment:
i don't think we need to specify `2`, you need composer. nobody should be using v1.

Reply:
Agreed. Removed.